lib/modules/platform/bitbucket-server/index.ts

```diff
@@ -721,14 +730,17 @@ async function updatePRAndAddReviewers(
     logger.warn({ err, reviewers, prNo }, `Failed to add reviewers`);
     if (err.statusCode === 404) {
       throw new Error(REPOSITORY_NOT_FOUND);
-    } else if (
-      err.statusCode === 409 &&
-      !utils.isInvalidReviewersResponse(err)
-    ) {
-      logger.debug(
-        '409 response to adding reviewers - has repository changed?',
-      );
-      throw new Error(REPOSITORY_CHANGED);
+    } else if (err.statusCode === 409) {
+      if (utils.isInvalidReviewersResponse(err) && !bitbucketInvalidReviewers) {
+        // Retry again with invalid reviewers being removed
+        const invalidReviewers = utils.getInvalidReviewers(err);
+        await updatePRAndAddReviewers(prNo, reviewers, invalidReviewers);
```

Comment:
I think going with an approach like this would be more concise:

```suggestion
        const filteredReviewers = reviewers.filter(
          (name) => !invalidReviewers.includes(name)
        );
        await updatePRAndAddReviewers(prNo, reviewers);
```